main/src/main/scala/sbt/plugins/SemanticDbPlugin.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt
+package plugins
+
+import Keys._
+import sbt.librarymanagement.syntax._
+import sbt.librarymanagement.CrossVersion
+import Project.inConfig
+import sbt.io.IO
+
+object SemanticDBPlugin extends AutoPlugin {
```

Comment:
By convention we would write `SemanticdbPlugin` in code even if in written form it's "SemanticDB". For some reason I think `SemanticdbPlugin` reads nicer than `SemanticDBPlugin`. The same applies for setting names, `enableSemanticdb`, `semanticdbIncludeInJar`, `semanticdbVersion`, `semanticdbCompilerPlugin`

Reply:
Sounds good.